recipe/build.sh

```diff
@@ -2,6 +2,12 @@
 
 set -euxo pipefail
 
+if [[ $target_platform == osx-arm64 ]]; then
+	cross_args="-DSTACK_DIRECTION=-1"
+else
+	cross_args=""
```

Comment:
```suggestion
```suggestion
    CMAKE_ARGS="${CMAKE_ARGS} -DSTACK_DIRECTION=-1"
```
```

Reply:
```suggestion
    CMAKE_ARGS="${CMAKE_ARGS} -DSTACK_DIRECTION=-1 -DHAVE_CLOCK_GETTIME=0 -DHAVE_CLOCK_GETTIME=0"
```

Reply:
> Suggested change 
> 	cross_args="-DSTACK_DIRECTION=-1"
> else
> 	cross_args=""
>     CMAKE_ARGS="${CMAKE_ARGS} -DSTACK_DIRECTION=-1"

Much cleaner, didn't think of this